### PR TITLE
Create a resource with craft command - Add/10

### PR DESCRIPTION
**Pull request by girardinsamuel**

### CODE REVIEW

File: src/masonite/api/providers/ResourceProvider.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class ResourceProvider(ServiceProvider):
+    """Provides Services To The Service Container."""
+
+    wsgi = False
+
+    def register(self):
+        """Register objects into the Service Container."""
+        # self.app.bind('ResourceCommand', ResourceCommand())
```
Comment (girardinsamuel) [inline src/masonite/api/providers/ResourceProvider.py:14]: From other existing commands I have chosen `self.commands` to register the command but the documentation does not specify that but instead the commented line `self.app.bind`.
Could someone confirm, that `self.commands` is the way to go ?

Comment (josephmancuso) [reply src/masonite/api/providers/ResourceProvider.py:14]: yes thats the way to go

Comment (josephmancuso) [reply src/masonite/api/providers/ResourceProvider.py:14]: https://docs.masoniteproject.com/architectural-concepts/service-providers#commands

**Comment by girardinsamuel** [comment]
This package has now been moved into Masonite core, I am closing this PR.

**Pull request closed without merging**